src/components/main/MainContent.jsx

```diff
@@ -0,0 +1,51 @@
+function MainContent() {
+  return (
+    <div>
+      <div className="text__lg text__center">
+        <div className="new__note">
+          <button className="btn btn__primary">Create a New Note</button>
+        </div>
+        <div className="note__input">
+          <input type="text" placeholder="Title" className="input__title" />
+          <textarea
+            name=""
+            id=""
```

Comment:
id and name if not used you should remove it.

Reply:
okay will do it